### PR TITLE
Glass Mapper C# Script File Additions

### PR DESCRIPTION
**Pull request by sitecoremaster**

### CODE REVIEW

File: src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx [hunk 1]
```diff
@@ -1,48 +1,49 @@
-﻿// Generates Sitecore Habitat-style constants classes
+﻿// Generates Glass Constants File
```
Comment (blipson89) [inline src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx]: I understand why you renamed the original `constants.csx` to `habitat.csx`, but I'm not sure what the purpose of this glass constants file is

Comment (sitecoremaster) [reply src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx]: The reason for the glass mapper specific constant (which I could combine into the main GlassMapper.csx) is to provide the Template Id (as an ID or as a String) and the Field Id's and Names to the attributes in the models.  Without this information, Glass would not always map to the correct field.

Ex.

`[SitecoreField(FieldName` = NameOfFieldFromConstantsFileAsAString)]

Same goes for the SitecoreType attribute which is used on the Class definition.

Comment (blipson89) [reply src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx]: Why not bake it right into the template like the Synthesis one does: https://github.com/blipson89/Leprechaun/blob/master/src/Leprechaun.CodeGen.Roslyn/Scripts/Synthesis.csx#L47

Comment (sitecoremaster) [reply src/Leprechaun.CodeGen.Roslyn/Scripts/Constants.csx]: I think that makes sense.  I think I got too caught up into separating the code out, but ultimately they will all appear in one file anyways.  So I will make that change later today.

**Comment by blipson89** [comment]
This has been merged into the dev branch

**Pull request merged**